src/components/App/App.js

```diff
@@ -95,12 +96,14 @@ export default function App() {
   )
 
   function safeCostsToHistory({ date, costs, keynote, urls }) {
+    const usedServices = services.filter(service => service.hours >= 1)
```

Comment:
Statt `service.hours >= 1` zu schreiben könntest du überlegen folgendes zu schreiben:
```suggestion
    const usedServices = services.filter(service => service.hours > 0)
```
Damit machst du direkt klar, dass du alles meinst was nicht null ist. Kleinigkeit aber scheint mir weniger Denkarbeit zu erfordern.

Reply:
Check!